blockchains/optimism/assets/0x50Bce64397C75488465253c0A034b8097FeA6578/info.json

```diff
@@ -1,10 +1,10 @@
 {
     "name": "HanChain",
-    "type": "OPTIMISM",
```

Comment:
What's the differences between these changes, the `type` are still `OPTIMISM`, you just changed the ordering?

Reply:
I've changed the `type` from `OPTIMISM` to `OPMAINNET`, but the `merge-fee-bot` automatically fixed my changes. Could you please fix it?
https://github.com/trustwallet/assets/pull/25547/commits/e9f090c4b663e22e677f7821b40ad97682a19dc1